nextjs-logs-manager/app/api/services/logHandler.ts

```
@@ -0,0 +1,36 @@
+   * @returns Array actualizado de logs
+   */
+  static updateLog(id: string, updates: Partial<LogEntry>): LogEntry[] {
+    const currentLogs = getLogs()
+    const updatedLogs = currentLogs.map(log => 
+      log.id === id ? { ...log, ...updates } : log
+    )
+    localStorage.setItem('logs', JSON.stringify(updatedLogs))
+    window.dispatchEvent(new Event('logsUpdated'))
+    return updatedLogs
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)